### PR TITLE
Update Pylint version.

### PR DESCRIPTION
**Pull request by peter-fogg**
@edx/ecommerce

### CODE REVIEW

**Comment by rlucioni** [comment]
:+1:

**Comment by rlucioni** [comment]
FYI @AlasdairSwan, making a change like this one in https://github.com/edx/programs will fix the failures you saw earlier today.

**Comment by peter-fogg** [comment]
FYI @renzo you may want to look again; upgrading seems to have introduced a few new violations to worry about which I've fixed.

File: ecommerce/extensions/checkout/signals.py [hunk 1]
```diff
@@ -9,7 +10,6 @@
 from ecommerce.extensions.checkout.utils import get_provider_data
 from ecommerce.notifications.notifications import send_notification
 from ecommerce.settings import get_lms_url
-from oscar.core.loading import get_class
```
Comment (rlucioni) [inline ecommerce/extensions/checkout/signals.py:12]: Did pylint catch this?

File: pylintrc [hunk 2]
```diff
@@ -7,9 +7,6 @@
 # pygtk.require().
 #init-hook=''
 
-# Profiled execution.
-profile=no
```
Comment (rlucioni) [inline pylintrc:11]: Why are lines in this file being removed?

File: ecommerce/extensions/checkout/signals.py (hunk 1 above)
Comment (peter-fogg) [inline ecommerce/extensions/checkout/signals.py:12]: Yes.

File: pylintrc (hunk 2 above)
Comment (peter-fogg) [inline pylintrc:11]: The removed lines are deprecated options which will be removed in Pylint 1.6.

**Comment by clintonb** [comment]
:+1:

File: ecommerce/extensions/checkout/signals.py (hunk 1 above)
Comment (rlucioni) [inline ecommerce/extensions/checkout/signals.py:12]: That's awesome! We finally don't need to call these out on PRs. I'm so happy.

**Comment by rlucioni** [comment]
:+1:

**Pull request merged**